datastream-server-restli/src/test/java/com/linkedin/datastream/server/TestCoordinator.java

```diff
@@ -92,7 +92,7 @@
 public class TestCoordinator {
   private static final Logger LOG = LoggerFactory.getLogger(TestCoordinator.class);
   private static final long WAIT_DURATION_FOR_ZK = Duration.ofMinutes(1).toMillis();
-  private static final int WAIT_TIMEOUT_MS = 30000;
+  private static final int WAIT_TIMEOUT_MS = 60000;
```

Comment:
Curious if this is happening because of any changes on the product side?

Reply:
Yes, there is newly added test(testCoordinationWithPartitionAssignment) may get timed out in a slow testing environment.